Do I follow JS best practices?

Mhdez221993 opened the pull request:
Please verify check my code and let me know if do I follow the best JS practices

Thank you.

Mhdez221993 committed:
Add all content of to do list in one file to be reviewd

Mhdez221993 committed:
Add html & css to be reveiwed

Mhdez221993 committed:
Add code to be reviewed

Mhdez221993 committed:
Made a copy of todo list

Mhdez221993 committed:
Merge branch 'master' into best-practices

Mhdez221993 committed:
Update readme

CollinsTatang reviewed (commented):
Hello Hernandez,

Your project is good and working as requested but:
Please can you see how you can work on the live demo link for it to display the working part of the project, not just the readme file

CollinsTatang reviewed (changes requested):
**CHANGES REQUIRE** 
Hello Hernandez,

Your project is good and working as requested but:
Please can you see how you can work on the live demo link for it to display the working part of the project, not just the readme file
Happy Codding

Mhdez221993 committed:
Fix gitpages

Mhdez221993 committed:
O
Merge branch 'best-practices' of github.com:Mhdez221993/Do-I-follow-JS-best-practices into best-practices

Mhdez221993 committed:
Add live demo link

Mhdez221993 committed:
Fix double string to single

Mhdez221993 commented:
@CollinsTatang  Thank you for reviewing my project here is the fixed link to my deployed project
https://mhdez221993.github.io/Do-I-follow-JS-best-practices/

CollinsTatang reviewed (approved):
**APPROVE**
Nice job Hernandez, your project has been approved you can now merge the branch to the main.
Happy Coding Brother.
